server/plugins/auth/openid.ts

```diff
@@ -0,0 +1,27 @@
+import log from "../../log";
+import Config from "../../config";
+import type {AuthHandler} from "../auth";
+
+const openIDAuth: AuthHandler = (manager, client, user, _, callback) => {
+	if (user === "") {
+		log.error(`Authentication failed`);
+		return callback(false);
+	}
+
+	// If no user is found, create it
+	if (!client) {
```

Comment:
don't call this client unless it is our irc client, which I guess it isn't.
Using terms used in a specific context all over the place as something else should be avoided